Sources/SnapshotTestCase.swift

```diff
@@ -92,8 +92,15 @@ open class SnapshotTestCase : XCTestCase {
     }
     
     private func recordSnapshot(of snapshotable: Snapshotable, options: Options, functionName: String, file: StaticString, line: UInt) throws {
-        try coordinator.recordSnapshot(of: snapshotable, options: options, functionName: functionName, line: line)
-        XCTFail("🔴 RECORD MODE: Reference image saved.", file: file, line: line)
+        let path = try coordinator.recordSnapshot(of: snapshotable, options: options, functionName: functionName, line: line)
+        XCTFail("🔴 RECORD MODE: Reference image saved to \(path.absoluteStringWithoutScheme)", file: file, line: line)
```

Comment:
I think you can solve this without your helper method by doing something like this:
```suggestion
        let snapshotUrl = try coordinator.recordSnapshot(of: snapshotable, options: options, functionName: functionName, line: line)
        XCTFail("🔴 RECORD MODE: Reference image saved to \(snapshotUrl.path)", file: file, line: line)
```